Nitrox.Launcher/ViewModels/LaunchGameViewModel.cs

```diff
@@ -242,6 +242,14 @@ private async Task StartSubnauticaAsync(string[]? args = null)
 
         // Start game & gaming platform if needed.
         IGamePlatform platform = GamePlatforms.GetPlatformByGameDir(subnauticaPath);
+        
+        // BIG PICTURE MODE: Two-stage launch process
+        if (keyValueStore.GetIsBigPictureModeEnabled() && platform is Steam steamPlatform)
+        {
+            await LaunchWithBigPictureMode(steamPlatform, subnauticaExe, subnauticaLaunchArguments);
+            return; // Exit early - Big Picture mode handles launch in background
+        }
```

Comment:
This could can be moved inside Steam.cs as it's unrelated to all other platforms anyway.